### PR TITLE
ci: add started_at and ended_at fields to TestJob

### PR DESCRIPTION
**Pull request by mwasilew**
Currently only LAVA backend implements filling in started_at and
ended_at fields. They can be empty for other backends.

Signed-off-by: Milosz Wasilewski <milosz.wasilewski@linaro.org>

### CODE REVIEW

**Comment by mwasilew** [comment]
@stevanradakovic, @mrchapp please check if this is helpful.

**Comment by stevanradakovic** [comment]
Sure is, thanks Milosz.

File: test/ci/backend/test_lava.py [hunk 1]
```diff
@@ -300,7 +300,26 @@
     'status': 'Complete',
     'id': 1234,
     'definition': yaml.dump(JOB_DEFINITION),
-    'multinode_definition': ''
+    'multinode_definition': '',
+    'start_time': '2020-11-13T10:23:50.019407Z',
+    'end_time': '2020-11-13T10:23:41.265697Z'
```
Comment (mrchapp) [inline test/ci/backend/test_lava.py:305]: Ends before starting?

**Review by mrchapp — COMMENTED** [review]
+1 Looks good to me, but someone else must approve.

File: test/ci/backend/test_lava.py (hunk 1 above)
Comment (mwasilew) [reply test/ci/backend/test_lava.py:305]: who knows, maybe it travels faster than light? :)

Comment (chaws) [reply test/ci/backend/test_lava.py]: `job name: back to the future`

**Comment by chaws** [comment]
LGTM! I'm amazed to see this number of missing target in testjob tests. Tests are failing, i guess it's a flake8 nit-pick

**Comment by mwasilew** [comment]
When you don't save the object to DB, there is no enforcement on the 'target'. So it used to work for tests. But not the object is saved early, so properties have to be adjusted. I'll take a look at the failure.

**Pull request merged**